recipes/conversation_view/feed_item_row/feed_item_row.stories.js

```diff
@@ -0,0 +1,144 @@
+import { action } from '@storybook/addon-actions';
+import { createTemplateFromVueFile, getIconNames } from '@/common/storybook_utils';
+import DtRecipeFeedItemRow from './feed_item_row';
+import DtRecipeFeedItemRowMdx from './feed_item_row.mdx';
+import DtRecipeFeedItemRowDefaultTemplate from './feed_item_row_default.story.vue';
+import DtRecipeFeedItemRowVariantsTemplate from './feed_item_row_variants.story.vue';
+
+const iconsList = getIconNames();
+
+export const argsData = {
+};
+
+/*
+  Controls
+  ========
+
+  Here we define any custom controls or control overrides for our components.
+
+  By default storybook will attempt to provide an appropriate control of the same name for each property in the
+  component as well as include any description provided using a prop decorator within your component (see above).
+
+  Storybook will also attempt to provide an appropriate control for each slot in the component as well as include any
+  description provided using a slot decorator within your component (see below).
+
+  <!-- @slot example slot decorator -->
+*/
+export const argTypesData = {
+  // Slots
+  /*
+    We use the following naming scheme `<SLOT_NAME>Slot` for slot controls to prevent conflicts with props that share
+    the same name. We provide the correct name of the slot using the name control attribute to ensure that the argument
+    table and description within the controls accurately reflects the correct names of our component's props and slots.
+  */
+  default: {
+    control: 'text',
+    table: {
+      type: {
+        summary: 'VNode',
+      },
+    },
+  },
+
+  threading: {
+    name: 'threading',
+    description: 'Slot for threading row component',
+    control: 'text',
+    table: {
+      category: 'slots',
+      type: {
+        summary: 'VNode',
+      },
+    },
+  },
+
+  reactions: {
+    name: 'reactions',
+    description: 'Slot for reactions row component',
```

Comment:
Instead of adding description you can add a comment in the vue template code like this right above the slot. Same for the other slots.

```
<!-- @slot Slot for reactions row component -->
<slot #reactions />
```